package.json

```diff
@@ -2,18 +2,34 @@
   "name": "emoji-inject-msteams",
   "version": "1.0.0",
   "description": "Injecting rich emoji capabilities into MSTeams",
-  "main": "Payload.js",
+  "main": "src/payload.js",
   "scripts": {
+    "build-ts": "tsc",
+		"watch-ts": "tsc -w",
     "test": "",
-    "install-local": "sh installer.sh",
-    "build-extension": "sh build-chrome-extension.sh"
+    "install-local": "sh scripts/installer.sh",
+    "build-extension": "sh scripts/build-chrome-extension.sh"
   },
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "jest": "^24.8.0"
+    "@types/jquery": "^3.5.6",
+    "cross-env": "^7.0.3",
+    "jest": "^24.9.0",
+    "nodemon": "^1.19.1",
+    "tslint": "^5.18.0"
   },
   "dependencies": {
-    "asar": "^2.0.1"
+    "@azure/storage-blob": "^12.0.0",
```

Comment:
why reference azure storage, is this a modification which stores emojis into azure storage directly?